chia/full_node/mempool_manager.py

```diff
@@ -350,6 +362,7 @@ async def validate_spend_bundle(
             new_spend: spend bundle to validate
             npc_result: result of running the clvm transaction in a fake block
             spend_name: hash of the spend bundle data, passed in as an optimization
+            first_added_height: The block height when this SpendBundle was first added
```

Comment:
I don't think this description is super helpful. The variable name says the same thing. My understanding is that this is the full node's peak height at the time this transaction was received from the network. Different nodes may have different heights for the same transactions (right?). It's not obvious what it's useful for, that would be nice to have an explanation for also, especially given that it's not a network-wide constant.

Reply:
Yes, I'll put some thought into a better explanation.